### PR TITLE
cleanup meta

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: packages/@ember/-internals/meta/lib/meta.ts [hunk 1]
```diff
@@ -516,50 +516,46 @@ export class Meta {
...
+  Meta.prototype.peekValues = function(key: string) {
+    return this._findInherited2('_values', key);
   };
 
-  Meta.prototype.deleteFromValues = function(subkey: string) {
-    delete this._getOrCreateOwnMap('_values')[subkey];
+  Meta.prototype.deleteFromValues = function(key: string) {
+    if (this._values !== undefined) {
+      delete this._values[key];
+    }
```
Comment (bekzod) [inline packages/@ember/-internals/meta/lib/meta.ts]: avoid creating `_values` map only to delete its prop

File: packages/@ember/-internals/meta/lib/meta.ts [hunk 2]
```diff
@@ -516,50 +516,46 @@ export class Meta {
...
   };
 
-  Meta.prototype.readInheritedValue = function(key, subkey) {
-    let internalKey = `_${key}`;
-
+  Meta.prototype.readInheritedValue = function(key) {
```
Comment (bekzod) [inline packages/@ember/-internals/meta/lib/meta.ts]: remove extra key which is always the same `_values`

**Review by rwjblue — COMMENTED** [review]
Looks like we still have a `_bindings: any | undefined` on the class

**Comment by krisselden** [comment]
Is this all the way done?

**Pull request closed without merging**